onnxruntime/core/mlas/lib/tanh.cpp

```diff
@@ -119,6 +119,11 @@ Return Value:
 
         float Value = *Input++;
 
+	// This odd "if test" exists to ensure an input value of NaN carries through 
```

Comment:
>	 [](http://example.com/codeflow?start=0&length=1)

nit: space

Reply:
I'm sorry, I get you are saying "nit" as in "this is a small complaint" but when you say "space," I do not understand what you are requesting...

Reply:
This line used a tab and the others used spaces.  When I edit it, it looks fine.
I will change it to use spaces as well.